docs-2.0/3.ngql-guide/10.tag-statements/3.alter-tag.md

```diff
@@ -14,7 +14,9 @@
 ALTER TAG <tag_name>
     <alter_definition> [[, alter_definition] ...]
     [ttl_definition [, ttl_definition] ... ]
-    [COMMENT = '<comment>'];
+    [COMMENT '<comment>']
+    [NOT NULL]
```

Comment:
NULL | NOT NULL?

Sometimes we may wanna change from NOT NULL back to NULL.

Reply:
default value is null. so I have declared it could be not null also.